### PR TITLE
Check graph sync status

### LINKED ISSUE #728: Check graph sync status on asset details

### PR DESCRIPTION
**Pull request by mihaisc**
Fixes #728

Added errors in Web3Feedbak and removed them from asset page 

image

image

### CODE REVIEW

File: src/hooks/useGraphSyncStatus.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export function useGraphSyncStatus(): UseGraphSyncStatus {
   const { block, web3Loading } = useWeb3()
   const [blockGraph, setBlockGraph] = useState<number>()
   const [blockHead, setBlockHead] = useState<number>()
-  const [isGraphSynced, setIsGraphSynced] = useState(true)
+  const [isGraphSynced, setIsGraphSynced] = useState(false)
```
Comment (kremalicious) [inline src/hooks/useGraphSyncStatus.ts:62]: won't this result in the faulty graph sync status showing up all the time first, and then disappearing? Should be the other way around, should only come up when there is an error detected, not by default

Comment (mihaisc) [reply src/hooks/useGraphSyncStatus.ts:62]: no, this was just for testing and forgot to change it, good catch

File: src/components/molecules/Web3Feedback.tsx [hunk 2]
```diff
@@ -18,24 +19,29 @@ export default function Web3Feedback({
...
+      : 'warning'
 
   const title = !accountId
     ? 'No account connected'
     : // : !ocean
     // ? 'Error connecting to Ocean'
     accountId && isAssetNetwork === false
     ? 'Not connected to asset network'
+    : isGraphSynced === false
+    ? `The data for this network has only synced to Ethereum block ${blockGraph} (out of ${blockHead}). Please check back soon.`
```
[the remark this answers is not in the document]
Comment (mihaisc) [reply src/components/molecules/Web3Feedback.tsx:44]: fair enough, will move this into the description. If i take another look i kinda swapped the title and the description

**Comment by mihaisc** [comment]
Messages fixed
image

File: src/components/App.tsx [hunk 3]
```diff
@@ -40,17 +40,17 @@ export default function App({
...
   const { accountId } = useWeb3()
   const { isInPurgatory, purgatoryData } = useAccountPurgatory(accountId)
-  const { isGraphSynced, blockHead, blockGraph } = useGraphSyncStatus()
+  // const { isGraphSynced, blockHead, blockGraph } = useGraphSyncStatus()
 
   return (
     <Styles>
       <div className={styles.app}>
-        {(props as PageProps).uri.includes('/asset') && !isGraphSynced && (
+        {/* {(props as PageProps).uri.includes('/asset') && !isGraphSynced && (
```
Comment (kremalicious) [inline src/components/App.tsx:48]: so far the place we put the warning seems prominent enough so we should not need all the commented out stuff in here, so safe to just remove

**Review by kremalicious — APPROVED** [review]
perfect, all good. Only that minor removal of commented out code in `src/components/App.tsx` would be cool before merging

**Pull request merged**